test_scripts/ATF_ShutDown_MASTER_RESET.lua

```diff
@@ -0,0 +1,147 @@
+--  Requirement summary:
+--  Check that SDL finish its work properly MASTER_RESET with one registered App
```

Comment:
@LitvinenkoIra, this should be in section "-- Description:"

For requirement summary, you can use as example: 
"[Policies] Master Reset"